website/app/templates/show.hbs

```diff
@@ -14,7 +14,7 @@
     <DynamicTemplate @templateString={{this.renderedContent}} @componentId={{this.model.id}} />
 
     {{#if this.relatedComponents}}
-      <Doc::RelatedComponents @cards={{this.relatedComponents}} />
+      <Doc::RelatedComponents @cards={{this.relatedComponents}} @title={{this.title}} />
```

Comment:
Nitpick: I think `@component` or `@componentTitle` would work better as argument name because
- it's what we're technically passing
- will be more explicit what we're concatenating at line 20 of the `website/app/components/doc/related-components/index.hbs` file